testutils/keys/keys.go

```diff
@@ -0,0 +1,121 @@
+/*
+   Copyright 2018-2019 Banco Bilbao Vizcaya Argentaria, S.A.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package keys
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"io/ioutil"
+	"math/big"
+	"net"
+	"os"
+	"time"
+)
+
+func GenerateSignKey(path string) (string, error) {
+	/* public, private, err := ed25519.GenerateKey(nil)
+	if err != nil {
+		return "", err
+	} */
+
+	err := ioutil.WriteFile(path+"/id_ed25519", []byte(`-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW
+QyNTUxOQAAACCXiy9taNh+avmEAyzfnlCXxYHzgS3wGPCi52AY8qQyTwAAAKA6Sq8iOkqv
+IgAAAAtzc2gtZWQyNTUxOQAAACCXiy9taNh+avmEAyzfnlCXxYHzgS3wGPCi52AY8qQyTw
+AAAEAzpsL9rtrmKhL3cEHFcKPEvkd8y/QJXeFTtyhgaYfUDpeLL21o2H5q+YQDLN+eUJfF
+gfOBLfAY8KLnYBjypDJPAAAAHWdkaWF6bG9AbWFjc2xhY2suc2xhY2t3YXJlLmVz
+-----END OPENSSH PRIVATE KEY-----
+`), 0644)
+	if err != nil {
+		return "", err
+	}
+	err = ioutil.WriteFile(path+"/id_ed25519.pub", []byte("ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJeLL21o2H5q+YQDLN+eUJfFgfOBLfAY8KLnYBjypDJP gdiazlo@macslack.slackware.es"), 0644)
```

Comment:
we want your "email" publicly available ,-) we can use something else as name for the public key, right?

Reply:
This will be changed soon enough, when we remove the ssh dependency and provide a cli to generate the keys and a csr.